server/core/templates/cred_update/add_password_modal_partial.html

```diff
@@ -6,37 +6,70 @@ <h5 class="modal-title" id="staticPasswordLabel">Add a New Password</h5>
                 <button type="button" class="btn-close" aria-label="Close"></button>
             </div>
             <div class="modal-body">
-                <form class="row g-3 needs-validation" novalidate
-                      hx-post="/ui/reset/pw_check"
-                      hx-trigger="input from:#new-password delay:200ms changed, input from:#new-password-check delay:200ms changed">
-                    <input hidden=true type="text" autocomplete="username" />
+                <form class="row g-3 needs-validation" id="newPasswordForm" novalidate
```

Comment:
`needs-validation` is from the [bootstrap example site](https://getbootstrap.com/docs/5.0/forms/validation/#custom-styles), if we aren't referencing it, we don't need it.

```suggestion
                <form class="row g-3" id="newPasswordForm" novalidate
```